packages/orbit-components/src/Slider/index.tsx

```diff
@@ -287,14 +232,14 @@ const PureSlider = ({
     }
   };
 
-  const handleMouseDown = (i: number) => (event: MouseEvent) => {
+  const handleMouseDown = (i: number) => (event: React.MouseEvent<HTMLDivElement>) => {
```

Comment:
Could you please adapt `handleMouseMove` types as well?

Reply:
The thing is that some events get passed to `.addEventListener` while others are react event handlers. That’s the reason why I needed to keep this one as a bare event.

Reply:
Got it.